test/config/tslint-custom-rules-with-package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "rulesDirectory": ["tslint-test-custom-rules/rules"]
```

Comment:
I tried to add a test, but didn't use `"test/external/tslint-test-custom-rules/rules"`, because it would throw. I'm not sure, this works now as you expected...?